### PR TITLE
Docs/better contracts

### PR DESCRIPTION
**Pull request by ZaguePrime**

### CODE REVIEW

File: docs/api-contracts/Disputes.md [hunk 1]
```diff
@@ -112,4 +104,14 @@ type DisputeCreateResponse = string;
...
 
 **NOTE:** This endpoint involves files being exchanged. This requires the use of `multipart/form-data` instead of JSON.
-This is left incomplete until this exception is documenteda.
\ No newline at end of file
+This is left incomplete until this exception is documenteda.
+
+# Dispute Status Change
+- **Endpoint:** `PUT /dispute/status_update`
+- **Headers:**
+    - `Authorization: Bearer <>JWT`
```
Comment (ghost) [inline docs/api-contracts/Disputes.md]: Very nit, but shouldnt JWT be inside the brackets?

File: docs/api-contracts/Disputes.md [hunk 2]
```diff
@@ -112,4 +104,14 @@ type DisputeCreateResponse = string;
...
+```ts 
+type updateRequest = {
```
Comment (ghost) [inline docs/api-contracts/Disputes.md]: More nit: should be `PascalCase`

File: docs/api-contracts/Disputes.md [hunk 3]
```diff
@@ -112,4 +104,14 @@ type DisputeCreateResponse = string;
     - `Authorization: Bearer <JWT>`
 
 **NOTE:** This endpoint involves files being exchanged. This requires the use of `multipart/form-data` instead of JSON.
-This is left incomplete until this exception is documenteda.
\ No newline at end of file
+This is left incomplete until this exception is documenteda.
+
+# Dispute Status Change
+- **Endpoint:** `PUT /dispute/status_update`
```
Comment (ghost) [inline docs/api-contracts/Disputes.md]: To maintain consistency, shouldn't it rather be `update-status`? Or even just `status`, considering the *update* part is encoded by using the `put` method

File: docs/api-contracts/Disputes.md [hunk 4]
```diff
@@ -81,24 +81,16 @@ type DisputeResponse = {
...
-  };
+```json
+{
+  "title": "string",
+  "description": "string",
+  "respondent[full_name]": "string",
+  "respondent[email]": "string",
+  "respondent[telephone]": "string",
+  "files": "all the files"
 }
```
Comment (ghost) [inline docs/api-contracts/Disputes.md]: Please make this a typescript definition for consistency. For files, just denote it as `File`

**Review by ghost — CHANGES_REQUESTED** [review]
See my comments (not the nitpicking ones)

File: docs/api-contracts/Disputes.md (hunk 4 above)
Comment (ZaguePrime) [reply docs/api-contracts/Disputes.md]: please review change

File: docs/api-contracts/Disputes.md [hunk 5]
```diff
@@ -82,15 +82,15 @@ type DisputeResponse = {
...
-  "files": "all the files"
-}
+```ts
+type DisputeCreateRequest = {
+  title : string;
+  description : string;
+  respondent[full_name] : string;
+  respondent[email] : string;
+  respondent[telephone] : string;
+  files : File
```
Comment (Vafdaf12) [inline docs/api-contracts/Disputes.md:92]: Don't we have multi-file upload support?

**Review by ghost — APPROVED** [review]

**Pull request merged**